src/test/java/com/google/sps/DestinationDataServletTest.java

```diff
@@ -0,0 +1,90 @@
+package com.google.sps;
+
+import static com.google.appengine.api.datastore.FetchOptions.Builder.withLimit;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.verify;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.gson.Gson;
+import com.google.sps.servlets.Constants;
+import com.google.sps.servlets.DestinationDataServlet;
+import java.io.IOException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+@RunWith(JUnit4.class)
+public final class DestinationDataServletTest {
+  @Mock private HttpServletRequest request;
+  @Mock private HttpServletResponse response;
+
+  private DestinationDataServlet servlet;
+
+  private static final String NAME_PARAMETER = "name";
```

Comment:
Instead of redefining these constants, you could make the corresponding fields public in DestinationDataServlet and then just use them like  DestinationDataServlet.NAME_PARAMETER